src/charts/bar.js

```diff
@@ -141,13 +155,16 @@ export default function Bar (_container) {
       .append("g")
       .attr("class", "bar-group")
       .merge(stackedBarGroups)
-      .attr("fill", (d) => scales.colorScale(d.key))
       .attr("stroke", "white")
 
     stackedBarGroups.exit().remove()
 
     const stackedBars = stackedUpdate.selectAll(".mark")
-      .data((d) => d)
+      .data((d, i, p) => {
```

Comment:
we should also apply this logic to the regular bars right?

Reply:
For the stacked bar chart, the simple bar chart is a special case of `drawStackedBars`. `drawBars` is used by `groupedBar` and combo chart, which doesn't need this feature yet and may have different requirements

Reply:
ah I did not realize that, sounds good